lib/rule-tester/rule-tester.js

```diff
@@ -400,6 +404,46 @@ class RuleTester {
         this[IT] = value;
     }
 
+    /**
+     * Adds the `only` property to a test to run it in isolation.
+     * @param {string | ValidTestCase | InvalidTestCase} item A single test to run by itself.
+     * @returns {ValidTestCase | InvalidTestCase} The test with `only` set.
+     */
+    static only(item) {
+        if (typeof item === "string") {
+            return { code: item, only: true };
+        }
+
+        return { ...item, only: true };
+    }
+
+    static get itOnly() {
+        if (typeof this[IT_ONLY] === "function") {
+            return this[IT_ONLY];
+        }
+        if (typeof this[IT] === "function" && typeof this[IT].only === "function") {
+            return Function.bind.call(this[IT].only, this[IT]);
+        }
+        if (typeof it === "function" && typeof it.only === "function") {
+            return Function.bind.call(it.only, it);
+        }
+
+        if (typeof RuleTester[DESCRIBE] === "function" || typeof RuleTester[IT] === "function") {
```

Comment:
```suggestion
        if (typeof this[DESCRIBE] === "function" || typeof this[IT] === "function") {
```

For consistency with other usages of these properties, and might matter if `RuleTester` was extended.

Reply:
> might matter if RuleTester was extended

Actually, I'm not sure why we use `this` in static accessors, but at the end call `RuleTester.describe()` and  `RuleTester.it()` directly on `RuleTester`.

Reply:
Good point, `this` is odd here. I'll commit this suggestion for consistency. Good eye.